lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2460,7 +2463,7 @@ def stroke(self):
         """
         # _linewidth > 0: in pdf a line of width 0 is drawn at minimum
         #   possible device width, but e.g., agg doesn't draw at all
-        return (self._linewidth > 0 and self._alpha > 0 and
+        return (self.get_linewidth() > 0 and self._alpha > 0 and
```

Comment:
Is this change related? I don't see any change to `get_linewidth` or `_linewidth` that might require it.

Reply:
Not really, I just felt accessing a private variable using the _getter_ would be more appropriate